docs/source/conf.py

```diff
@@ -6,12 +6,12 @@
 # -- Project information -----------------------------------------------------
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
 
-import instrument
+import bits
 
-project = "instrument"
+project = "BITS"
 copyright = "2023-2025, APS BCDA"
 author = "APS BCDA"
-version = instrument.__version__
+version = bits.__version__
```

Comment:
Should also add `bits.__version__` to the metadata configured in `utils/metadata.py`